frontend/src/views/files/Editor.vue

```diff
@@ -148,8 +147,7 @@ export default {
 
       this.$store.commit("updateRequest", {});
 
-      let uri = url.removeLastDir(this.$route.path) + "/";
-      this.$router.push({ path: uri });
+      history.back();
```

Comment:
this.$router.push({ path: uri }) will add a new route link to the route history. We should use history.back() here. Then when we click the "back" button in the browser, we will not revisit this .txt file. the windows explorer works in the same way.